test/e2e/docker/Dockerfile

```diff
@@ -1,10 +1,18 @@
 # We need to build in a Linux environment to support C libraries, e.g. RocksDB.
 # We use Debian instead of Alpine, so that we can use binary database packages
-# instead of spending time compiling them.
+# instead of spending time compiling them. 
+# -> There is currently no librocksdb-dev v6.17.x that is necessary by line/gorocksdb.
+# So we download rocksdb and build it.
+
 FROM golang:1.15
 
 RUN apt-get -qq update -y && apt-get -qq upgrade -y >/dev/null
-RUN apt-get -qq install -y libleveldb-dev librocksdb-dev libtool >/dev/null
+RUN apt-get -qq install -y libleveldb-dev make libc-dev libtool >/dev/null
+WORKDIR /
+RUN wget -O rocksdb-v6.20.3.tar.gz https://github.com/facebook/rocksdb/archive/v6.20.3.tar.gz
+RUN tar -zxvf rocksdb-v6.20.3.tar.gz
+RUN cd rocksdb-6.20.3 && make -j2 shared_lib && make install-shared
```

Comment:
I'm not sure that:
* do ostracon depends on `rocksdb`?
* how to build rocksdb on local?

Reply:
In local, we don't need to consider rocksdb. e2e docker image needs all db backend to test.

Reply:
If you want to use rocksdb, then you can just type `make build OSTRACON_BUILD_OPTIONS=rocksdb`.

Reply:
It's easy to use `rocksdb`! Thanks.